src/scalar_mul/straus.rs

```diff
@@ -12,11 +12,16 @@
 
 #![allow(non_snake_case)]
 
+#[cfg(any(feature = "alloc", feature = "std"))]
```

Comment:
Why not guard the entire `use straus;` in the `scalar_mul/mod.rs`, or maybe the whole `use scalar_mul`?

Reply:
We can't guard the entire `use scalar_mul` since we want to be able to do some scalar muls with no_std, but the guards could probably be cleaner.